Jint/Options.cs

```diff
@@ -385,6 +386,11 @@ public class ConstraintOptions
         /// </summary>
         public int MaxRecursionDepth { get; set; } = -1;
 
+        /// <summary>
+        /// Maximum recursion stack count, defaults to -1 (as-is dotnet stacktrace).
+        /// </summary>
+        public int MaxExecutionStackCount { get; set; } = StackGuard.Disabled;
```

Comment:
I think the old comment had valuable info `14000 which comes from Chrome and V8 based engines (ClearScript) that can handle 13955.`  So might be good to indicate that option will allow more nested calls with slight performance / stack trace readability drawback. (after hitting the engine's own limit),

Reply:
I updated it as `remarks`. If you have another suggestion, please share it with me. It will be a pleasure for me to improve it.